### DISCUSSION TITLE
[Question][Lecture 11-2] About abstract semantics of conditional

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#611] [Q&A]
![image](https://user-images.githubusercontent.com/71373857/120275555-766f2d00-c2ec-11eb-80cf-73ab363a5c47.png)

In the lecture, we learned like the figure above about the abstract semantics of conditionals. But I think that if f<sub>E</sub>(m#) is Bottom, m# should be changed to m#{x->Bottom}. Am I understanding something wrong? Thank you

**Comment by KihongHeo** [comment]
Nice catch. You are right.
Since the deadline has been already passed, we won't consider such cases for evaluation.

**Comment by ee12ha0220** [comment]
Thank you!
